soca/obs/icefb_c2_esa.yaml

```diff
@@ -0,0 +1,40 @@
+- obs space:
+    name: icefb_c2_esa
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/icefb_c2_esa.{{window_begin}}.nc4
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/$(experiment).icefb_c2_esa.{{window_begin}}.nc4
+    simulated variables:  [sea_ice_freeboard]
+  obs operator:
+    name: SeaIceThickness
+  obs error:
+    covariance model: diagonal
+  #_letkf: &letkf
+  #  # note, this is only used for LETKF. If running with LETKF, the workflow
+  #  # will append "<< : *letkf" to the end of this file
+  #  obs localizations:
+  #  - localization method: Horizontal Gaspari-Cohn
+  #    lengthscale: 500e3
+  obs filters:
+  - *obs_land_mask
+  - filter: Bounds Check
+    minvalue: -0.1
+    maxvalue: 1.0
+  - filter: Domain Check
+    where:
+    - variable: { name: sea_surface_temperature@GeoVaLs}
+      maxvalue: -0.5
```

Comment:
I wonder if we should impose sst to be even closer to freezing, -1.5? That might help removing obs that may be too close to the seaice edge? Something to keep in mind.

Reply:
I will add this as part of the calibration test parameter